mira/dkg/web_client.py

```diff
@@ -72,8 +72,7 @@ def web_client(
 
     res.raise_for_status()
 
-    if res.text and res.json():
-        return res.json()
```

Comment:
Changed this since it returned None for e.g., a valid but empty list JSON but it's possible I missed some corner case in which a string is returned that isn't valid JSON.